README.md

```diff
@@ -121,7 +126,8 @@ a(); // Object -- so far so good.
 a().getA(); // "a"
 ```
 
-Yes. Got it. In both of these instances, we actually created a brand new object, and then immediately threw it away, because we didn't assign it to anything. Don't worry about that.
+Yes. Got it. In both of these instances, we actually created a brand new object, and then immediately threw it away, 
+because we didn't assign it to anything. Don't worry about that.
```

Comment:
And it is not different. I made it less different!

I made this change to avoid horizontal scrolling when I read the .md file.

It's what Markdown was designed for - to be comfortable to read as both text and HTML.